projectDocs/issues/triage.md

```diff
@@ -0,0 +1,178 @@
+## Intent
+This page intends to outline some of the information that might be helpful to those trying to triage issues.
+
+Most of the issues raised on the NVDA GitHub repository fall into one of the following categories:
+
+- adding support to new applications
+- adding new features in NVDA
+- adding new features to support 3rd party applications better
+- bug reports
+
+Firstly we want to catch high priority issues and ensure that they are attended to first.
+This might be things like:
+
+- a crash of NVDA
+- synthesisers not working correctly
+- an error in an existing feature.
+
+Secondly, we want to ensure that there is enough information on the issue so that it can be well understood and work can start when it comes to the front of the queue. The sooner we do this, the more likely it is that we will get the information we need.
+Most of the information required is asked for in the Github issue templates, so this is a great place to start.
+
+## Check for duplicates
+Pick a few keywords and search the NVDA repository on Github. We can also check if there is already a pull request in the works that may fix this issue.
+
+## What kind of issue is it?
+Can we label this as a regression, a requested change of behaviour, or a requested new feature?
+
+### Regression

```

Comment:
May be distinguish between regressions in NVDA or regressions in essential applications due to updates (i.e. browsers, windows applications, etc.)